EIPS/eip-2357.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2357
```

Comment:
Not sure where 2357 came from, but that seems be something else: https://github.com/ethereum/EIPs/pull/2357

I suggest to use 2622 which is the number of this PR>